myproject/myapp2/management/commands/create_client.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
+
+from myproject.myapp2.models import Client
```

Comment:
Нужно так
```suggestion
from .models import Client
```
Должно работать

Reply:
я написал как вы предложили, но пайчарм всё равно пишет ошибку:    
from .models import Client
 
ModuleNotFoundError: No module named 'myapp2.management.commands.models'

Reply:
А вы в каждой команде поменяли?

Reply:
Да. В каждом из CRUD который я написал ( во всех 5-и файлах). Мне тоже это кажется странным так как в лекции и на практике таких проблем с этим не было.